content/amp_amg/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Monitoring using Amazon Managed Service for Prometheus / Grafana"
+chapter: true
+weight: 53
+pre: '<i class="fa fa-film" aria-hidden="true"></i> '
+---
+
+# Configure Amazon Managed Service for Prometheus / Grafana for your ECS cluster
+
+### Introduction
+
+In this chapter we will learn about setting up Monitoring for the your ECS environment using Amazon Managed Service for Prometheus / Grafana and collecting metrics using AWS Distro for OpenTelemetry.
+
+
+#### Amazon Managed Service for Prometheus (AMP)
+Amazon Managed Service for Prometheus is a monitoring service for metrics compatible with the open source Prometheus project, making it easier for you to securely monitor container environments. AMP is a solution for monitoring containers based on the popular Cloud Native Computing Foundation (CNCF) Prometheus project. AMP is powered by Cortex, an open source CNCF project that adds horizontal scalability to ingest, store, query, and alert on Prometheus metrics. AMP reduces the heavy lifting required to get started with monitoring applications across Amazon Elastic Kubernetes Service and Amazon Elastic Container Service, as well as self-managed Kubernetes clusters. AMP automatically scales as your monitoring needs grow. It offers highly available, multi-Availability Zone deployments, and integrates AWS security and compliance capabilities. AMP offers native support for the PromQL query language as well as over 150+ Prometheus exporters maintained by the open source community.
+
+
+{{% button href="https://aws.amazon.com/prometheus/faqs/" icon="fab fa-leanpub" icon="fab fa-leanpub" icon-position="right"  %}}Learn more about AMP{{% /button %}}
+
+
+
+#### Amazon Managed Service for Grafana (AMG)
```

Comment:
The formatting on this page is a bit funky. Can you take a look and clean up pls?